src/themes/variables.js

```diff
@@ -50,6 +50,7 @@ const Variables = css`
         --color-grey-30: #f5f7fa;
         --color-grey-31: #f8fafc;
         --color-grey-32: #dcdee0;
+        --color-pale-gray-64: rgba(245, 247, 250, 0.64);
```

Comment:
the equivalent hex for this is #f5f7fa which is --color-grey-30

Reply:
Yes, thanks Yashim, I ended up using `--color-grey-31` which was a match.